test/functional/Valhalla/playlist.xml

```diff
@@ -49,6 +50,7 @@
 			<subset>Valhalla</subset>
 		</subsets>
 		<impls>
+			<impl>hotspot</impl>
```

Comment:
I'll try to figure out a new way to do this. For now, being able to run with hotspot manually is good enough